scripts/pylib/twister/twisterlib/reports.py

```diff
@@ -275,6 +275,9 @@ def json_report(self, filename, version="NA", platform=None):
             if used_rom:
                 suite["used_rom"] = used_rom
 
+            if self.env.options.enable_size_report:
+                suite["rom_ram_from_buildlog"] = self.env.options.footprint_from_buildlog
```

Comment:
the new property name in twister.json is chosen to be aligned with the related properties there (`used_rom`, `used_ram`, `available_rom`, `available_ram`).